subworkflows/local/mtbseq-nf-modes/normal_analysis.nf

```diff
@@ -42,5 +44,16 @@ workflow NORMAL_ANALYSIS {
                  params.user,
                  references_ch)
 
+        ch_versions = ch_versions.mix(TBGROUPS.out.versions)
+        ch_multiqc_files.mix(TBFULL.out.statistics)
+        ch_multiqc_files.mix(TBFULL.out.classification)
+        ch_multiqc_files.mix(TBGROUPS.out.distance_matrix)
+        ch_multiqc_files.mix(TBGROUPS.out.groups)
```

Comment:
The important change here was that, reassignment was necessary i.e. after the initialization of channels with `Channel.empty()`, we need to reassign and only then use operators like `mix`.